tcf_website/static/reviews/new_review.js

```diff
@@ -124,8 +124,48 @@ jQuery(function($) {
                 $("#semester").prop("disabled", false);
             });
     });
+
+    // Code for the progress bar
+    $("#reviewtext").keydown(function() {
+        // Used .trim() to remove leading and trailing spaces
+        var review = $("#reviewtext").val().trim();
+        var numberOfWords = countNumberOfWords(review);
+
+        // Set the width of the bar to the what percent of 300 words the current review is
+        $("#reviewprogressbar").width(1000 * (numberOfWords / 300));
+        if (numberOfWords < 100) {
```

Comment:
When I was testing the progress bar, it seemed the color of the bar was the same both when the word count was less than 100 and between 100 and 200. I feel like it should change colors once the user passes 100 words as the text does so that their progress in writing the review is further enforced?

Reply:
It does change colors from red to orange but I think you just caught a bug where if you delete everything, the color stays the same

Reply:
Yeah I see from the code it does seem to change colors, but I think the coloring was so light the change was practically unnoticeable? Might be worth making more distinct if possible. It looks like you updated the colors now and the change is definitely noticeable now which is good. Looks like you fixed the bug where it didn't update immediately after deleting (vice versa, it also updates immediately when you undo the deletion) so great job! I love how dynamic it is.